app/controllers/HomeController.scala

```diff
@@ -57,28 +57,25 @@ class HomeController @Inject()(val homeView: views.html.Home,
 
   private lazy val errorHandler: Boolean => ShowInternalServerError = (isAgent: Boolean) => if (isAgent) itvcErrorHandlerAgent else itvcErrorHandler
 
-  private def view(nextPaymentDueDate: Option[LocalDate], overDuePaymentsCount: Option[Int], nextUpdatesTileViewModel: NextUpdatesTileViewModel,
-                   paymentCreditAndRefundHistoryTileViewModel: PaymentCreditAndRefundHistoryTileViewModel, dunningLockExists: Boolean, currentTaxYear: Int,
-                   displayCeaseAnIncome: Boolean, isAgent: Boolean, origin: Option[String] = None)
-                  (implicit user: MtdItUser[_]): Html =
-    homeView(
-      origin = origin,
+  private def view(nextUpdatesTileViewModel: NextUpdatesTileViewModel, nextPaymentsTileViewModel: NextPaymentsTileViewModel, returnsTileViewModel: ReturnsTileViewModel,
```

Comment:
I am not sure if we need this method anymore, this can be done within calling method